akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -116,17 +116,6 @@ class TestKit(system: ActorSystem) {
    */
   def unwatch(ref: ActorRef): ActorRef = tp.unwatch(ref)
 
-  /**
-   * Ignore all messages in the test actor for which the given partial
-   * function returns true.
-   */
-  def ignoreMsg(pf: JFunction[Any, Boolean]): Unit = {
```

Comment:
this is still ok though, since `f` returns Boolean

Reply:
Ups, sorry, looked for docs mentioning PF but didn't look at the actual signature.